rest/endpoints/user_recordings/profile_recordings.yml

```diff
@@ -71,9 +77,48 @@ paths:
           example: 1526648593
           schema:
             type: integer
+        - in: query
+          name: filter[starts_after]
+          required: false
+          description: Include only recordings having broadcast start time after specified time (in Unix timestamp).
+          example: 1526648593
+          schema:
+            type: integer
+        - in: query
+          name: filter[ends_before]
+          required: false
+          description: Include only recordings having broadcast end time before specified time (in Unix timestamp).
+          example: 1526648593
+          schema:
+            type: integer
+        - in: query
+          name: filter[ends_after]
+          required: false
+          description: Include only recordings having broadcast end time after specified time (in Unix timestamp).
+          example: 1526648593
+          schema:
+            type: integer
+        - in: query
+          name: filter[series_id]
+          required: false
+          description: Consider only recordings having the same series_id.
+          example: "abdf8785-cb23-445b-946f-69e8770500a1"
+          schema:
+            type: string
+            nullable: true
+        - in: query
+          name: filter[channel_id]
```

Comment:
Hmm, I wonder since here we're filtering on attributes of a related resource (broadcast_slot), whether the filter should instead be `filter[broadcast_slot.channel.id]`